core/templates/dev/head/pages/exploration_editor/exploration_metadata_form_directive.html

```diff
@@ -0,0 +1,73 @@
+<p ng-class="{'has-error': !explorationTitleService.displayed.length > 0}" ng-style="pStyle">
+  <label ng-style="formStyle">{{metadataFormTitle}}</label>
+  <input id="explorationTitle" type="text" class="form-control protractor-test-exploration-title-input" ng-model="explorationTitleService.displayed" ng-blur="saveExplorationTitle()" placeholder="Choose a title for your exploration." focus-on="<[::EXPLORATION_TITLE_INPUT_FOCUS_LABEL]>" maxlength="40" ng-trim="false">
```

Comment:
Where are EXPLORATION_TITLE_INPUT_FOCUS_LABEL, saveExplorationTitle(), etc. defined? May need to pass them in explicitly via the 'scope' field, so that the directive is self-contained. Ditto elsewhere.